corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -378,6 +387,14 @@
       {% endcompress %}
     {% endif %}
 
+    {% if request.use_datetimepicker and not requirejs_main and use_bootstrap5 %}
```

Comment:
Why `not requirejs_main`? Is it because these are included explicitly in requirejs pages?

Why not have that condition on the CSS include above?

Reply:
the javascript in `requirejs` pages is included by specifying `datetimepicker` in the list of dependencies. There is no way to import css through `requirejs` dependencies, as `requirejs` is only responsible for javascript bundling. This is how it's done for all the libraries that have a css component

Reply:
Makes sense.

> There is no way to import css through `requirejs` dependencies...

Vellum [does it](https://github.com/dimagi/Vellum/blob/9d2f2db789028bd2901ddfdc676c0b28879165a4/src/main.js#L82) ([with less too](https://github.com/dimagi/Vellum/blob/master/src/core.js#L35)). I forgot that we did not have that capability in HQ.